tiled/client/base.py

```diff
@@ -261,12 +262,12 @@ def metadata_copy(self):
         ]  # returning as list of mutable items
 
     @property
-    def specs(self):
+    def specs(self) -> ListView[Spec]:
         "List of specifications describing the structure of the metadata and/or data."
         return ListView([Spec(**spec) for spec in self._item["attributes"]["specs"]])
 
     @property
-    def access_blob(self):
+    def access_blob(self) -> DictView[Any, Any]:
```

Comment:
This can also be constrained to be [JSON](https://github.com/bluesky/tiled/blob/406ea84e33b177a070bb02361c6f2c90c0546bdc/tiled/type_aliases.py#L15).